convert2rhel/breadcrumbs.py

```diff
@@ -69,7 +69,7 @@ def collect_early_data(self):
         self._set_started()
 
     def finish_success(self):
-        """Set data accessible after sucessfull conversion and generate the JSON file."""
+        """Set data accessible after successfully conversion and generate the JSON file."""
```

Comment:
This should be `Set data accessible after a successful conversion` or `Set data accessible after successfully converting`. Now it's a mix of a past and future action